app/assets/components/tagify/.bower.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
@leoapost interesting idea. the thing is that tags, in this case, are postcodes so it's not behaving as a common "tags". They have they own model. Another thing is that the validation is running on front end not after you save it so we would still have to change the js. And the last thing is that this is actually replacing old js plugin that was there. So I'm not introducing new dependency I'm just updating old one and isn't gem dependency too ;)